seed/seedingScript.js

```diff
@@ -18,8 +18,19 @@ try {
   const mainImagesStream = fs.createWriteStream(filePath + '/mainImages.csv');
   generateCSV(mainImagesStream, createMainImageRecords, 1e7, () => {
     mainImagesStream.end();
-    console.log('Successfully wrote mainImage records to CSV')
+    console.log('Successfully wrote mainImage records to CSV');
   });
 } catch (err) {
   if (err) {console.log(err)}
 }
```

Comment:
well here it is again, are you throwing something else somewhere? I think the err first if statement only applies with call backs, in the try/catch scheme its an error in the catch block, right?

Reply:
yeah i think youre right about this one. i definitely just threw it in there to have something in the catch block. good call! i will update